pages/src/search/search.ts

```diff
@@ -0,0 +1,26 @@
+import {ESLBaseElement} from '../../../src/modules/esl-base-element/core';
+import {CSSClassUtils} from '../../../src/modules/esl-utils/dom/class';
+import {ready} from '../../../src/modules/esl-utils/decorators/ready';
+import {loadSearchScript} from './search-script';
+
+export class ESLDemoSearchPageWrapper extends ESLBaseElement {
+  static is = 'esl-d-search-page-wrapper';
+
+  @ready
+  protected connectedCallback(): void {
+    loadSearchScript();
+    this.initSearchScript();
+  }
+
+  private initSearchScript(): void {
+    (window as any).__gcse = {
+      parsetags: 'onload',
+      initializationCallback: (): void => this.afterSearchScriptLoad()
+    };
+  }
+
+  private afterSearchScriptLoad(): void {
+    const loadingAnimationEL = this.querySelector('.animation-loading')!;
```

Comment:
Please remove strict type disabling.
The `Element.[proto].querySelector` returns Element | null.
CSSClassUtils.add expect Element (not nullable).
Typescript says "here is a "null pointer"-like problem, please make your script safe",
Do not ignore that, better fix it.